tests/performance-test/grafana/perftest-dashboard.json

```diff
@@ -1068,7 +1070,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "collectd_total_amqp_processed_message_count",
+          "expr": "sg_total_metric_rcv_count",
```

Comment:
As a metric this name does make a lot more sense. I'm glad you made the decision not to relabel the metrics